### PR TITLE
REWORK: conflict section for record-types rewritten

### PR DESCRIPTION
**Pull request by pawel-kow**
Said rework to the conflict chapter.

### CODE REVIEW

File: Domain Connect Spec Draft.adoc [hunk 1]
```diff
@@ -1583,29 +1586,34 @@ display to them the specific records that are being set in DNS.
...
 
-The proposed handling of records and conflicts is as follows (if not
-otherwise specified, conflicts occur if the records have the same name):
+Conflict detection done by DNS provider prior to template application has to take
+into consideration specifics of each DNS record type. The rules outlined below shall
+assure predictable conflict resolution between DNS providers. Each rule applies to
+the records on the very same host, unless specifed otherwise.
+
+* CNAME records conflicts with all other records
+* any record type conflicts with existing CNAME record
```
Comment (arnoldblinn) [inline Domain Connect Spec Draft.adoc]: Not sure this line makes  sense

File: Domain Connect Spec Draft.adoc [hunk 2]
```diff
@@ -1583,29 +1586,34 @@ display to them the specific records that are being set in DNS.
...
+** None: This indicates that the TXT records do not conflict with any other TXT
+record. This is the default setting, if not specified.
+** All: This indicates that the TXT records conflict with any other TXT record
+** Prefix: This indicates that TXT record conflict with any other TXT containing value starting with txtConfictMatchingPrefix 
+
+=== Re-apply
+
+When applying the same template again, identified by the same providerId, serviceId,
+domain and host, ideally DNS Providers would not detect the conflict. Instead, the records
+created by the initial template application would be removed and the new instance applied.
```
Comment (arnoldblinn) [inline Domain Connect Spec Draft.adoc]: I might add something that indicates this is optional and up to the DNS Provider.

Comment (pawel-kow) [reply Domain Connect Spec Draft.adoc]: added additional language in 6d45f5dfe78776e4b0c2d279dffe982e383a36b3

File: Domain Connect Spec Draft.adoc (hunk 1 above)
Comment (arnoldblinn) [reply Domain Connect Spec Draft.adoc]: I understand what you are saying; the wording is a bit offputting.

I would suggest: CNAME records conflict  with all other records, and any other records of any type conflict with an existing CNAME record.

Comment (pawel-kow) [reply Domain Connect Spec Draft.adoc]: Updated as per suggestion in afcef45c002de75af61dba5cb6c71ae08092cd71

**Pull request merged**